backend/inspirehep/records/views.py

```diff
@@ -48,6 +55,31 @@ def get(self, pid, record):
         return jsonify(data)
 
 
+class LiteratureReferencesResource(MethodView):
+    view_name = "literature_references"
+
+    @pass_record
+    def get(self, pid, record):
+        page = request.values.get("page", 1, type=int)
+        size = request.values.get("size", 25, type=int)
+
+        if page < 1 or size < 1:
+            abort(400)
+
+        if size > current_app.config["MAX_API_RESULTS"]:
+            raise MaxResultWindowRESTError()
```

Comment:
I am wondering if we actually want to throw an error to the web page or overwrite `size` with `current_app.config["MAX_API_RESULTS"]`, WDYT? 
Same above.

Reply:
we want to throw an error to the web page because this is what we (invenio actually) are doing for other search endpoints. So we want to keep the behavior consistent for all

Reply:
Also, I think the API user should be informed that he isn't getting all the results he asked for

Reply:
Eventually, would be the same for the check `< 1`.

Reply:
I don't think I follow. What do you mean? Ofc it is easy to add a check if it exceeds the ```MAX_API_RESULTS``` and overwrite it but this isn't what we want since this is inconsistent with all our other search endpoints

Reply:
That sounds reasonable